TypeCobol.Test/Parser/Programs/Cobol85/UsageIsPointer.rdz.cbl

```diff
@@ -0,0 +1,42 @@
+       IDENTIFICATION DIVISION.
+       PROGRAM-ID. Pgm2503.
+       DATA DIVISION.
+       WORKING-STORAGE SECTION.
+       01 Var1 pointer.
+       01 Var2 usage is pointer.
+
+       01 Var3 pointer-32.
+       01 Var4 usage is pointer-32.
+
+      *Ok on a group
+       01 Group1 pointer-32.
+      *    Ok because group has a valid usage that apply to its children
+           05 Var5.
+      *    Ok same usage as parent
+           05 Var6 pointer-32.
+
+      *    Ko usage different than parent
+      *    TODO currently not check by our parser
+      *The specified"USAGE"was different from the "USAGE"specified at
+      *the group level.  The group "USAGE" was assumed for this item.
```

Comment:
Is there an issue for that ? If not, please create it.

Reply:
Issue #2524 created